sdk/nodejs/environment.ts

```diff
@@ -70,9 +74,11 @@ export class Environment extends pulumi.CustomResource {
             resourceInputs["name"] = args ? args.name : undefined;
             resourceInputs["organization"] = args ? args.organization : undefined;
             resourceInputs["yaml"] = args ? args.yaml : undefined;
+            resourceInputs["revision"] = undefined /*out*/;
```

Comment:
This is odd... why is it in the inputs section?

Reply:
I think this is fine, I'm just not used to seeing outputs only.

Reply:
I'm guessing because it's an output only property?